### PR TITLE
Fixed ES in perftest

### PR DESCRIPTION
**Pull request by csibbitt**
* Migrated from client cert to password auth
* Dropped certificate handling
  * we don't need to check certs for localhost
* Adjusted hostnames and other queries for ECK
* Bonus: repo renames in the README and whitespace extermination

### CODE REVIEW

**Comment by csibbitt** [comment]
[Test results of adding 2nd & 3rd ES node to existing STF](https://trello.com/c/aWghkE3Q/799-test-results-of-adding-2nd-3rd-es-node-to-existing-stf)

File: tests/performance-test/README.md [hunk 1]
```diff
@@ -19,15 +19,15 @@ grafana instance and one that executes the testing logic.
 ## Setup
 
 SAF must already be deployed including the default ServiceTelemetry example CR.
-A quick way to do this is using the `quickstart.sh` script in 
-`telemetry-framework/deploy/` directory to run SAF.	
+A quick way to do this is using the `quickstart.sh` script in
+`service-telemetry-operator/deploy/` directory to run SAF.
```
Comment (leifmadsen) [inline tests/performance-test/README.md]: s/SAF/STF/g

File: tests/performance-test/performance-test.sh [hunk 2]
```diff
@@ -266,25 +266,25 @@ while true; do
...
 
-            # # get number of events seen by elasticsearch
-            # get_es_event_count
+            # get number of events seen by elasticsearch
+            get_es_event_count
 
-            # # calulate success ratio
-            # SUCCESS_RATIO=$(( ES_EVENT_RECV_COUNT / NUM_COLLECTD_EVENTS)).$(( (ES_EVENT_RECV_COUNT * 100 / NUM_COLLECTD_EVENTS) % 100 ))
+            # calulate success ratio
+            SUCCESS_RATIO=$(( ES_EVENT_RECV_COUNT / NUM_COLLECTD_EVENTS)).$(( (ES_EVENT_RECV_COUNT * 100 / NUM_COLLECTD_EVENTS) % 100 ))
```
Comment (leifmadsen) [inline tests/performance-test/performance-test.sh]: huh, I learned something new today about bash syntax!

**Comment by csibbitt** [comment]
# Testing

```
$ ./grafana-launcher.sh
[...]
Graphing dashboard available at: 
grafana-service-telemetry.apps-crc.testing

$ cd ..
$ ./performance-test.sh -t tb -c 60 -h 1 -p 1000 -i 1 -n 1
[...]
*** Collecting test results ***
tar: Removing leading `/' from member names
EVENTS generated: 71, recieved by Elastic Search: 71, success rate: 1.0

*** Posting events results to Elastic Search ***

*** Test complete ***
```

image

None of these things worked before.

File: tests/performance-test/performance-test.sh (hunk 2 above)
Comment (csibbitt) [reply tests/performance-test/performance-test.sh]: Even after all these years it seems like every day is discover-a-feature-of-bash day.

**Pull request merged**